pkg/codegen/convert/plugin_mapper.go

```diff
@@ -93,6 +93,11 @@ func ProviderFactoryFromHost(host plugin.Host) ProviderFactory {
 type mapperPluginSpec struct {
 	name    tokens.Package
 	version semver.Version
+	// An optional list of providers this plugin can map to, only filled if GetMappings is implemented.
+	mappings []string
+	// Set to true once we've called GetMappings, mappings may still be nil after this if GetMappings wasn't
+	// implemented.ß
```

Comment:
Last character here was not intentional I believe

Reply:
Haha no, still getting used to this MacBook keyboard